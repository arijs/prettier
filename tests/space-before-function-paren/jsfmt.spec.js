run_spec(__dirname, { spaceBeforeFunctionParen: true }, [
  "babylon",
  "typescript"
]);
