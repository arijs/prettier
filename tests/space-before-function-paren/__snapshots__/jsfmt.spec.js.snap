// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint.js 1`] = `
/*eslint space-before-function-paren: "error"*/
/*eslint-env es6*/

function foo() {
    // ...
}

var bar = function() {
    // ...
};

var bar = function foo() {
    // ...
};

class Foo {
    constructor() {
        // ...
    }
}

var foo = {
    bar() {
        // ...
    }
};

var foo = async() => 1
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/*eslint space-before-function-paren: "error"*/
/*eslint-env es6*/

function foo() {
  // ...
}

var bar = function() {
  // ...
};

var bar = function foo() {
  // ...
};

class Foo {
  constructor() {
    // ...
  }
}

var foo = {
  bar() {
    // ...
  }
};

var foo = async () => 1;

`;
